test/import/test_wmd_importer.cpp

```diff
@@ -270,3 +271,135 @@ INSTANTIATE_TEST_SUITE_P(
+        ASSERT_TRUE(graph.isLocal(mirrorTopology));
+        ASSERT_TRUE(!graph.isLocal(masterTopology));
+        bool found = false;
+        // Mirror must exist in mirror range.
+        auto it = graph.getMirrorRange();
+        for (auto v = it.begin(); v != it.end(); v++) {
+          if (*v == mirrorTopology) {
+            found = true;
+          }
+        }
+        ASSERT_TRUE(found);
+        found = false;
+        auto mirror_master_array = graph.getLocalMirrorToRemoteMasterOrderedTable();
+        for (auto elem : mirror_master_array) {
+          if ((lift(elem, getMirror) == mirrorTopology) &&
+              (lift(elem, getMaster) == masterTopology)) {
+            found = true;
+          }
+        }
+        ASSERT_TRUE(found);
+      } else {
+        // If I don't have mirror, that could be because it is in local, or never be a destination
+        // from me.
+        if (graph.isLocal(mirrorTopology)) {
+          // If it is from me, it is in my master range.
+          bool found = false;
+          auto it = graph.getMasterRange();
+          for (auto v = it.begin(); v != it.end(); v++) {
+            if (*v == mirrorTopology) {
+              found = true;
+            }
+          }
+          ASSERT_TRUE(found);
+          found = false;
+          // In mirror to master, this should never exist
+          auto mirror_master_array = graph.getLocalMirrorToRemoteMasterOrderedTable();
+          for (auto elem : mirror_master_array) {
+            if ((lift(elem, getMirror) == mirrorTopology) ||
+                (lift(elem, getMaster) == masterTopology)) {
+              ASSERT_TRUE(false);
+            }
+          }
+        }
+      }
+
+      auto goldenEdgeIt = std::find(goldenEdges.begin(), goldenEdges.end(), dstTok);
+      EXPECT_NE(goldenEdgeIt, goldenEdges.end())
+          << "Unable to find edge with src_tok: " << srcTok << "\tand dst_tok: " << dstTok
+          << "\tat vertex: " << (vid - 1);
+      ET dumbEdge = ET{numVertices, numVertices};
+      graph.setEdgeData(eh, dumbEdge);
+      eData = graph.getEdgeData(eh);
+      EXPECT_EQ(eData.src, numVertices);
+      EXPECT_EQ(eData.dst, numVertices);
+    }
+  }
+  graph.deinitialize();
+}
+
+INSTANTIATE_TEST_SUITE_P(SmallFiles, MirrorDLCSRInitEdgeList,
+                         ::testing::Values(std::make_tuple("/pando/graphs/simple.el", 10)));
+// std::make_tuple("/pando/graphs/rmat_571919_seed1_scale10_nV1024_nE10447.el",
+// 1024)));
+
+INSTANTIATE_TEST_SUITE_P(
+    DISABLED_BigFiles, MirrorDLCSRInitEdgeList,
+    ::testing::Values(
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale11_nV2048_nE22601.el", 2048),
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale12_nV4096_nE48335.el", 4096),
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale13_nV8192_nE102016.el", 8192),
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale14_nV16384_nE213350.el", 16384)));
```

Comment:
When you delete DISABLED does this pass?

Reply:
Yes, but it takes a lot longer. Do you want me to enable them?